gov/proposals.py

```diff
@@ -10,11 +10,11 @@
 
 def get_next_governance_id():
     cfg.current_governance_id += 1
-    return cfg.current_governance_id
+    return cfg.current_governance_id -1 
 
 def get_next_budget_id():
     cfg.current_budget_id += 1
-    return cfg.current_budget_id
+    return cfg.current_budget_id -1
```

Comment:
This looks extremely weird

Reply:
Have changed this back and modified the config.ini starting # to -1 so that the first proposal is #0 instead of doing it the way I have above.